db/migrate/20191030093601_add_imported_to_revision.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddImportedToRevision < ActiveRecord::Migration[5.2]
+  def change
+    add_column :revisions, :imported, :boolean, default: false
```

Comment:
This query could be slow to run - hopefully it's not as I don't think Content Publisher db is that big yet but worth checking on integration as it's locker https://medium.com/braintree-product-technology/postgresql-at-scale-database-schema-changes-without-downtime-20d3749ed680#e4a5

Reply:
This migration completed in 0.0977s on integration.  I don't think we need to worry about the locking with this duration, especially if the deployment is done outside normal publishing hours.